llvm/utils/TableGen/Basic/SequenceToOffsetTable.h

```diff
@@ -162,7 +162,8 @@ class SequenceToOffsetTable {
 
   /// emit - Print out the table as the body of an array initializer.
   /// Use the Print function to print elements.
-  void emit(raw_ostream &OS, void (*Print)(raw_ostream &, ElemT)) const {
+  void emit(raw_ostream &OS,
+            function_ref<void(raw_ostream &, ElemT)> Print) const {
```

Comment:
So that I can use `std::bind` in the emitter to pass an additional parameter to the printer function.